### PR TITLE
Move source of truth for version numbers outside of Cocoapods

### PR DESCRIPTION
**Pull request by ajsecord**
This moves the source of truth for MDC-iOS' version number to a file called VERSION at the root of the repo. Since all manipulation of version numbers is done through `scripts/release/bump`, this change should be transparent.

This paves the way for a fix to #2441 and is probably more correct in general.

Note that now that we removed Podfile.lock files, there is no need to do the slow `pod install` step when bumping version numbers. Hooray!

### CODE REVIEW

**Comment by ianegordon** [comment]
Does it make sense to use PlistBuddy to bump our Info.plist files as well?

File: scripts/release/manage_pods.py [hunk 1]
```diff
@@ -104,6 +104,12 @@ def install_all_podfile_dirs(directory, fast_install, blacklist):
   for d in dirs:
     install_podfile_dir(d, fast_install)
 
+    # If `fast_install` is not specified, we will do `pod update` again and
+    # again, even though it is a global operation and not a per-Podfile
+    # operation. Disable it after the first invocation to save time.
+    if not fast_install:
+      fast_install = True
```
Comment (jverkoey) [inline scripts/release/manage_pods.py:111]: Yessss!

**Review by jverkoey — APPROVED** [review]

**Comment by ajsecord** [comment]
Note build breakage was from #2400, I believe, not this change...

**Comment by ajsecord** [comment]
Moved @ianegordon 's question to #2447.

**Pull request merged**